modules/webhooks/infrastructure/drizzle/webhook-log-repository-drizzle.ts

```
@@ -1,189 +1,278 @@
+	dependencies: WebhookLogRepositoryDependencies = {},
+): WebhookLogRepository {
+	const db = dependencies.db ?? defaultDb;
+	const logger: Logger = dependencies.logger ?? {
+		debug: (fields) => console.log("DEBUG", fields),
+		info: (fields) => console.log("INFO", fields),
+		warn: (fields) => console.log("WARN", fields),
+		error: (fields) => console.error("ERROR", fields),
+	};
+	const clock = dependencies.clock ?? { now: () => new Date() };
+
+	function mapWebhookLog(row: typeof webhookLogTable.$inferSelect): WebhookLog {
```

Biome (2.4.7)

[error] Excessive complexity of 15 detected (max: 12).

(lint/complexity/noExcessiveCognitiveComplexity)